src/shards/JLLEnvs.jl

```diff
@@ -120,9 +126,9 @@ function get_system_dirs(triple::String, version::VersionNumber=v"4.8.5")
 
     # download shards
     if haskey(ENV, "JULIA_CLANG_SHARDS_URL") && !isempty(get(ENV, "JULIA_CLANG_SHARDS_URL", ""))
-        @info "Downloading artifact($(gcc_info.id)) from $(gcc_info.url) ..."
+        @info "Downloading artifact($(gcc_info.id))"
     end
-    Artifacts.download_artifact(Base.SHA1(gcc_info.id), gcc_info.url, gcc_info.chk)
+    Artifacts.ensure_artifact_installed(get_gcc_shard_key(triple, version), ARTIFACT_TOML_PATH[]; platform=BINARY_BUILDER_PLATFORM)
```

Comment:
```suggestion
    Artifacts.ensure_artifact_installed(get_gcc_shard_key(triple, version), ARTIFACT_TOML_PATH[])
```

This is fine because the dummy argument `platform` is ignored(not used for selecting/filtering artifacts like it is used in `artifact_meta`, etc.) in the current impl of `ensure_artifact_installed`.

Reply:
I also hoped it would have works, but it defaults to `HostPlatform()`😢.
https://github.com/JuliaLang/Pkg.jl/blob/e31a3dc77201e1c7c469f6d4572c521f93fefb20/src/Artifacts.jl#L397-L402

Reply:
What I meant was to directly call this method `ensure_artifact_installed(name::String, meta::Dict, artifacts_toml::String;...)`